pypit/masterframe.py

```diff
@@ -104,18 +108,26 @@ def master(self):
           arc image
 
         """
-        # Load the MasterFrame if it exists and user requested one to load it
+        # (KBW) Not sure this is how it should be treated if loaded is
+        # being deprecated
         msframe, header, raw_files = self.load_master_frame()
         if msframe is None:
             return None
-        else:
-            # Prevent over-writing the master frame when it is time to save
-            self.settings['masters']['loaded'].append(self.frametype+self.setup)
-            # Hold it
-            self.msframe = msframe
-        # Return
+        self.msframe = msframe
         return msframe.copy()
 
+#        # Load the MasterFrame if it exists and user requested one to load it
```

Comment:
How is this being done now?

Reply:
The only difference from the uncommented and commented code is the setting line.  I've removed the use of 'loaded'.  I'll have to figure out how to put this back in if 'loaded' is still being used.